pkg/plugins/platform/local.go

```diff
@@ -186,6 +185,24 @@ func (l *LocalApp) ImportDB(imPath string) error {
 	return nil
 }
 
+// SiteStatus returns the current status of an application based on the web container.
+func (l *LocalApp) SiteStatus() string {
+	webContainer, err := l.FindContainerByType("web")
+	if err != nil {
+		return "not found"
```

Comment:
We should at least emit the error found here I'd say, otherwise we're just hiding the result with "not found". I'd be OK with a log.